README.md

```diff
@@ -42,60 +52,200 @@ use PinkCrab\Registerables\Post_Type;
 
 class Basic_CPT extends Post_Type {
 
-	public $key      = 'basic_cpt';
-	public $singular = 'Basic';
-	public $plural   = 'Basics';
+  public $key      = 'basic_cpt';
+  public $singular = 'Basic';
+  public $plural   = 'Basics';
 }
 ```
+ 
+[See full Post Type docs](docs/Post-Type.md)
+
+## Taxonomy
 
 Creates a flat taxonomy for the **Post** Post Type.
 
 ``` php
 use PinkCrab\Registerables\Taxonomy;
 
 class Basic_Tag_Taxonomy extends Taxonomy {
-	public $slug         = 'basic_tag_tax';
-	public $singular     = 'Basic Tag Taxonomy';
-	public $plural       = 'Basic Tag Taxonomies';
-	public $description  = 'The Basic Tag Taxonomy.';
-	public $hierarchical = false;
-	public $object_type = array( 'post' );
+  public $slug         = 'basic_tag_tax';
+  public $singular     = 'Basic Tag Taxonomy';
+  public $plural       = 'Basic Tag Taxonomies';
+  public $description  = 'The Basic Tag Taxonomy.';
+  public $hierarchical = false;
+  public $object_type = array( 'post' );
 }
 ```
 
-## Testing ##
+[See full Taxonomy Docs](docs/Taxonomy.md)
+
+## Meta Box
+
+Create a simple meta box as part of a post type definition.
+```php
+class My_CPT extends Post_Type {
+  public $key      = 'my_cpt';
+  public $singular = 'CPT Post';
+  public $plural   = 'CPT Posts';
+
+  public function meta_boxes( array $meta_boxes ): array {
+    $meta_boxes = MetaBox::side('my_meta_box')
+      ->label('My Meta Box')
+      ->view_template('path/to/view/template')
```

Comment:
Why does the metabox have a template? 

Maybe using $var_name instead of strings would be better. The same with the array below

Reply:
becuase they do, vanilla wp has a view callback 
function($post, $args){ echo $post->ID;}

this allows you to have the template separate and pass your args in, so its not all a mess